boilerplate.js

```diff
@@ -43,18 +59,21 @@ function prepare (callback) {
 
   var done = after(this._boilerplate.length, callback)
     , out  = this.boilerplateOut = {}
+    , self = this;
 
-  this._boilerplate.forEach(function (src) {
-    var callback = done
-    findDestination(src, function (err, dst) {
-      if (err)
-        return callback(err)
-
-      out[src] = out[path.basename(src)] = path.basename(dst)
+  this._boilerplate.forEach(function (src, index) {
+    adjustForLang(src, self.lang, function(src) {
+      self._boilerplate[index] = src;
```

Comment:
Hey Martin,

Er, no, this isn't for cachability or perf at all, it's just so later mappings in `getExerciseText()`, that will use the *actual* source file path (hence the possibly locale-aware one), work properly. I'm not caching anything as I `adjustForLang` all the time (and it's once-per-process anyway).